src/workerd/server/workerd.capnp

```diff
@@ -424,6 +424,9 @@ struct Worker {
   # Where should the global "fetch" go to? The default is the service called "internet", which
   # should usually be configured to talk to the public internet.
 
+  cacheOutbound @11 :ServiceDesignator = "cache";
```

Comment:
```suggestion
  cacheApiOutbound @11 :ServiceDesignator = "cache";
```

Let's be clearer that this is specifically about the Cache API, not any implicit caching.

Reply:
Giving this a default value of `cache` will mean that all existing configs that don't declare any such service will break. I think this should default to null, in which case the cache API is considered disabled.